main/src/ca/uwaterloo/flix/api/lsp/provider/completion/KeywordCompleter.scala

```diff
@@ -90,74 +90,74 @@ object KeywordCompleter {
   def getExprKeywords(range: Range): List[Completion] =
     List(
       // A
-      Completion.KeywordCompletion("and"         , range, Priority.Default),
+      Completion.KeywordCompletion("and", range, Priority.Default),
```

Comment:
I think this part is easier to read with the non-standard whitespace. Is this change a must-do setting or just configuring the autoformat button?

Reply:
I cannot find an option to turn it off.